admin/includes/functions/general.php

```diff
@@ -2135,6 +2135,43 @@ function zen_products_id_valid($products_id) {
     }
   }
 
+
+/*
+ *  Check if option name is not expected to have an option value (ie. text field, or File upload field)
+ */
+  function zen_option_name_no_values($option_name_id) {
```

Comment:
Does it make better sense to name this function `zen_option_name_NEEDS_no_values`? (caps only for emphasis) ... or something else slightly more semantic? When looking at it I find myself wondering if the name means "has no values?" or "needs no values?" or "should never have any values?", etc.